### ISSUE TITLE
Improve the output of an execution

### ISSUE THREAD

**Issue by febo** [febo/P45#1]
The output should include:
- [x] information of the dataset being used
- [ ] accuracy / confusion matrix for training set
- [x] validate the tree on a given test set
